examples/example_omb_csv_reader.py

```diff
@@ -0,0 +1,51 @@
+import logging
+from pathlib import Path
+from trajan.readers.omb import read_omb_csv
+# import xarray as xr
+# import trajan as ta
+# import matplotlib.pyplot as plt
+
+if __name__ == "__main__":
+    # logging.basicConfig(level=logging.DEBUG)
+    logging.basicConfig(level=logging.ERROR)
+
+    ####################
+    # example 1: default
```

Comment:
Keep both examples activated, see the other examples for how the sphinx-gallery format works: the %% and so on have special meaning for creating sections.